apps/aesophia/test/contracts/oracles.aes

```diff
@@ -4,9 +4,9 @@ contract Oracles =
   function registerOracle(acct : address,
                           sign : int,   // Signed account address
                           fee  : int,
-                          ttl  : int) : oracle(string, int) =
+                          ttl  : int) : oracle(int, int) = // oracle(string, int) =
```

Comment:
@UlfNorell 
Using the type oracle(string, int) crashes at contract run time.
I will investigate further.